static/docs/user-guide/contributing-documentation.md

```diff
@@ -0,0 +1,111 @@
+# Contributing to the Documentation
+
+We welcome any contributions to [DVC.org](https://github.com/iterative/dvc.org).
+
+## How to suggest a change
+
+Excluding very trivial changes, all contributions should be connected to an
+existing issue.
+
+Please search the [issue tracker](https://github.com/iterative/dvc.org/issues)
+before creating a new issue. This issue can be about a typographical error, an
+updation of the documentation or simply an improvement in generation of
+documentation.
+
+If you want to fix or implement it, please read a few paragraphs below to learn
+how to submit your changes.
+
+## Structure of the project
+
+* [Content](https://github.com/iterative/dvc.org/tree/master/static/docs) -
+  static Markdown files that are rendered dynamically in the browser. Merging
+  changes to these files into the master branch is enough to update the docs and
+  redeploy the website.
+
+* [Media](https://github.com/iterative/dvc.org/tree/master/static/img) - add new
+  images, gifs, diagrams here. Reference them from the Markdown files like this:
+  `![](/static/img/reproducibility.png)`.
+
+* [Sections](https://github.com/iterative/dvc.org/tree/master/src/Documentation/sidebar.json) - 
+  edit it to register a new section with the navigation menu.
+
+## Submitting changes
+
+In case of a minor change, you can use the **Edit on Github** button (at the
+left of the page) to fork the project, edit it in place and create a PR.
+Alternatively, create a fork, clone, edit it with your regular editor and submit
+a PR to the master branch. We'll review and merge it as soon as possible.
+
+Otherwise, please refer to the following procedure:
+
+* Open a new issue in the [issue
+  tracker](https://github.com/iterative/dvc.org/issues).
+* Setup the [development environment](#development-environment).
+* Fork [DVC.org](https://github.com/iterative/dvc.org.git) and prepare necessary
+  changes by referring to [structure of project](#structure-of-the-project).
+* Format the code by following the [code style guide](#code-style-guide).
+* Format JavaScript code styles by running `npm run prettier-src`.
```

Comment:
does it ensure 2 spaces, btw? have you tried it?

Reply:
@shcheklein I formatted it using Rewrap. On checking, there are two spaces.

Reply:
I mean the `npm run prettier-src`. I haven't tried it personally, that's why I'm asking )

Reply:
@shcheklein `prettier` runs only on JS files, right? This is a markdown file.
On a side note, I ran it and saw that a lot of files aren't following the `prettier` configuration.
![Screenshot from 2019-05-12 02-54-13](https://user-images.githubusercontent.com/35191225/57575063-50eefc80-7461-11e9-8058-8bb9698ec8e8.png)

Reply:
Yes, I got the idea. My question is - are default settings for the `prettier` aligned with the code style in the guide you put together. 2 spaces indent in JS - is one of them. I like the idea of introducing the `prettier` the same way we do with `black` in DVC repo. We can even use it as a hook potentially.

Reply:
@shcheklein 
```
semi: false
singleQuote: true
trailingComma: none
```
These are the configuration in the `.prettierrc` file. It doesn't seem to be updated.

Reply:
@shcheklein I think it will be nice to open an issue to update the `.prettierrc` file and add it as a pre-commit hook. What say?

Reply:
agreed! here is the link https://prettier.io/docs/en/precommit.html and check the DVC contrib guide's instruction on how black is used. Let's do this as a separate PR. I'm going to merge this one. It looks good to me already. When we add a hook we can update it if needed.